backend/hitas/views/reports.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+from typing import Any
+
+from django.http import HttpResponse
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.viewsets import ViewSet
+
+from hitas.services.apartment_sale import find_sales_on_interval
+from hitas.services.reports import build_sales_report_excel
+from hitas.types import HitasJSONRenderer
+from hitas.views.utils.excel import ExcelRenderer, get_excel_response
+
+
+class SalesReportSerializer(serializers.Serializer):
+    start_date = serializers.DateField()
+    end_date = serializers.DateField()
+
+    def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
+        if attrs["start_date"] > attrs["end_date"]:
+            raise serializers.ValidationError("Start date must be before end date")
+        return attrs
+
+
+class SalesReportView(ViewSet):
+    renderer_classes = [HitasJSONRenderer, ExcelRenderer]
+
+    def create(self, request: Request, *args, **kwargs) -> HttpResponse:
```

Comment:
Don't all of our other PDF endpoints use GET instead of POST? 🤔 I'd like to keep things consistent if possible :)

Reply:
This is an Excel endpoint, but `download-regulation-results` Excel endpoint also uses GET, so I can convert this to get for consistency. 

Although this endpoint has all of its parameters required when the other endpoint has only optional parameters which suit query parameters better..

Reply:
Done